data/releases.yml

```diff
@@ -30,7 +30,7 @@
         - name: sha512
           link: https://downloads.apache.org/skywalking/8.1.0/apache-skywalking-apm-8.1.0-src.tgz.sha512
   distribution:
-    - version: v8.3.0-es6
+    - version: v8.3.0 for ElasticSearch 6
```

Comment:
I knew this release works for `H2/MySQL/TiDB/InfluxDB` too, but this could be more clear, right?